src/modelfree/configs/multi/score.py

```diff
@@ -59,10 +59,22 @@ def adversary_fn(_env, _victim_index, _our_id, _opponent_id):
     return _gen_configs(victim_fn=_zoo_identity, adversary_fn=adversary_fn, envs=envs)
 
 
-def _adversary_vs_victims(adversary_type, adversary_paths, envs=None):
-    def adversary_fn(env, victim_index, our_id, _opponent_id):
+def _adversary_vs_victims(adversary_type, adversary_paths, envs=None, only_trained=False):
```

Comment:
I feel like `only_trained` is a mildly confusing parameter name because it's not that the other potential adversaries are untrained per se, they're just not trained against this victim. That said, I can't think of an obviously better name here, so take this as a possible suggestion rather than a strong one.

Reply:
I've renamed to `no_transfer`.